src/Nether.Analytics.GameEvents/ScoreEvent.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Nether.Analytics.GameEvents
+{
+    public class ScoreEvent : IGameEvent
+    {
+        public string Type => "score";
```

Comment:
We should probably put these type strings into a static class :-)
I'd also recommend that we prefix them with "nether-" to keep it clear which events are ones that we directly support (and avoid conflicts)